lib/packwerk/version.rb

```diff
@@ -1,6 +1,17 @@
-# typed: strong
+# typed: true
```

Comment:
Had to drop thiis down because I can't add sorbet in here cleanly, it break `gem install` during CI